### PR TITLE
Add pdcurses/3.9 recipe

### PR DESCRIPTION
**Pull request by madebr**
Specify library name and version:  **pdcurses/3.9**

- [X] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [X] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/pdcurses/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+        tools.save(os.path.join(self.package_folder, "licenses", "LICENSE"), self._license_text)
+
+        if self.settings.os == "Windows":
+            self.copy(pattern="curses.h", src=self._source_subfolder, dst="include")
+            self.copy(pattern="*.dll", dst="bin", keep_path=False)
+            self.copy(pattern="*.lib", dst="lib", keep_path=False)
+            self.copy(pattern="*.a", dst="lib", keep_path=False)
+
+            if self.settings.compiler != "Visual Studio":
+                os.rename(os.path.join(self.package_folder, "lib", "pdcurses.a"),
```
Comment (uilianries) [inline recipes/pdcurses/all/conanfile.py]: why does os.rename should work for self.options.shared?

Comment (madebr) [reply recipes/pdcurses/all/conanfile.py]: Can you rephrase your question? I don't understand it :smile:

Comment (danimtb) [reply recipes/pdcurses/all/conanfile.py]: ping @uilianries

Comment (madebr) [reply recipes/pdcurses/all/conanfile.py]: This renaming only happens on Windows.
On Windows, even dll's have a static (imp) library.

Comment (uilianries) [reply recipes/pdcurses/all/conanfile.py]: 5 days after my comment, I can't realize what I tried to say ... Anyway, it looks good.
